code/src/Form.js

```diff
@@ -0,0 +1,258 @@
+import React, { useState } from 'react';
```

Comment:
And this file has a mixture of semicolons. Either include them everywhere or not at all :)